jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunServiceActor.scala

```diff
@@ -77,6 +81,9 @@ class JobRunServiceActor(
   }
 
   def startJobRun(jobRun: JobRun): StartedJobRun = {
+    jobsRunningMetric.increment()
```

Comment:
I'm wondering if this is misleading... the job at this point goes to the queue... it isn't running... and it is "started".  thoughts?

Reply:
Should we just replace `running` to `active` in the metric name?